src/mapproxyUtils/mapproxySeed.ts

```diff
@@ -382,4 +394,41 @@ export class MapproxySeed {
       throw err;
     }
   }
+
+  private async handleInvalidBboxError(task: ISeed, jobId: string, taskId: string, attempt = 1): Promise<void> {
+    if (attempt > this.invalidBboxRetryLimit) {
+      this.logger.error({
+        msg: `Exceeded max retries (${this.invalidBboxRetryLimit}) for invalid bbox error, aborting seed operation for task ${taskId}`,
+        taskId,
+        layerId: task.layerId,
+        jobId,
+        maxRetries: this.invalidBboxRetryLimit,
+      });
+      throw new ExceededMaxRetriesError(`Exceeded max retries (${this.invalidBboxRetryLimit}) for invalid bbox error`);
+    }
+
+    const currentBuffer = this.invalidBboxInitialBufferMeters * attempt;
+
+    this.logger.warn({
+      msg: `Invalid bbox detected for geometry, applying ${currentBuffer}m buffer and retrying seed operation`,
+      taskId,
+      layerId: task.layerId,
+      bufferMeters: this.invalidBboxInitialBufferMeters,
+      originalGeometry: task.geometry,
+      errorType: 'invalid_bbox',
+    });
+
+    try {
+      const bufferedPolygon = buffer(task.geometry as Feature<Polygon>, currentBuffer, { units: 'meters' });
+      if (bufferedPolygon !== undefined) {
+        const bufferedTask = { ...task, geometry: bufferedPolygon };
+        await this.writeGeojsonTxtFile(this.geometryCoverageFilePath, JSON.stringify(bufferedPolygon), jobId, taskId);
+        await this.executeSeed(bufferedTask, jobId, taskId);
+      }
+    } catch (err) {
+      if (err instanceof Error && err.message.match(this.invalidBboxErrorPattern)) {
+        await this.handleInvalidBboxError(task, jobId, taskId, attempt + 1);
+      }
```

Comment:
missing throw error in case of any other error while buffered succeed in any followed attempt